packages/@sfx/sayt-driver-plugin/test/unit/browser/sayt-driver-plugin.test.ts

```diff
@@ -0,0 +1,193 @@
+import { expect, spy, stub } from '../../utils';
+import { SaytDriverPlugin } from '../../../src/index';
+import { DomEventsPlugin } from '../../../../dom-events-plugin/src/index';
+import { SaytPlugin } from '../../../../sayt-plugin/src';
+
+describe('Sayt Driver Plugin', () => {
+  let Driver: any;
```

Comment:
Use a lowercase `driver`. This is not a class. This can also be implicitly typed, so you don't need to type it.
```suggestion
  let driver;
```